src/mvAppItem.cpp

```diff
@@ -4234,6 +4355,10 @@ DearPyGui::GetEntityParser(mvAppItemType type)
         args.push_back({ mvPyDataType::UUID, "colormap", mvArgType::KEYWORD_ARG, "0", "mvPlotColormap_* constants or mvColorMap uuid from a color map registry" });
         args.push_back({ mvPyDataType::Float, "min_scale", mvArgType::KEYWORD_ARG, "0.0", "Sets the min number of the color scale. Typically is the same as the min scale from the heat series." });
         args.push_back({ mvPyDataType::Float, "max_scale", mvArgType::KEYWORD_ARG, "1.0", "Sets the max number of the color scale. Typically is the same as the max scale from the heat series." });
+        args.push_back({mvPyDataType:: String, "format", mvArgType::KEYWORD_ARG, "'%g'", "Formatting used for the labels."});
+        args.push_back({mvPyDataType:: Bool, "invert", mvArgType::KEYWORD_ARG, "False", "invert the colormap bar and axis scale (this only affects rendering; if you only want to reverse the scale mapping, make scale_min > scale_max)"});
```

Comment:
`invert` - I think we could come up with a better name for this, like `top_down` (since that's exactly what it does - instead of rendering the axis bottom-up, it renders it in the downwards direction), or `reverse`, `reverse_dir`, `reverse_axis`. The name `invert`, when used on a color widget, is a bit misleading since one could expect it to invert colors, not the direction. I like `top_down` better, but some day it can become problematic if all of a sudden ImPlot implements a horizontal colormap scale. So maybe `reverse_dir` after all?